src/_ert/forward_model_runner/fm_step.py

```diff
@@ -76,7 +76,7 @@ def killed_by_oom(pids: Sequence[int]) -> bool:
     return False
 
 
-class Job:
+class FMstep:
```

Comment:
Maybe a nitpick, but I think the S should be big too?

Reply:
It probably should. Potentially go all the way and write it as ForwardModelStep? That object name is actually not in use anywhere else in the codebase (although we do use it as a prefix for some other object names)

Reply:
ForwardModelStep sounds better to me